xml/deployment_preparation.xml

```diff
@@ -173,38 +179,65 @@
-    pre-installed operating is configured to look for a medium with the
-    volume label <literal>cidata</literal>. Configuration files on this
-    volume are searched and applied by <command>cloud-init</command>.
-   </para>
-   <para>
-    Read the following sections for more details. For further details,
-    refer to <xref linkend="sec.deploy.cloud-init"/>.
-   </para>
-   <para>
-    You need to build at least two ISO images with configuration
-    information: one for the &admin_node; and one for the
-    &worker_node;s. If you wish to assign specific host names or
-    customize individual &worker_node;s, then you should create
-    separate ISO images for each node.
+    while the node is starting up. For this purpose, &productname;
+    includes <command>cloud-init</command>. The described process is
+    very similar for all hypervisors. The examples in this manual use
+    &kvm; running on &sle;. The following procedure outlines the
+    overall process.
    </para>
+   <procedure>
+    <step>
+     <para>
+      For &vmware; ESX/ESXi: Convert VMDK images. See <xref linkend=
+      "sec.deploy.preparation.disk_images.vmware" />.
+     </para>
+    </step>
+    <step>
+     <para>
+      Write <command>cloud-init</command> configuration files. See
+      <xref linkend="sec.deploy.preparation.disk_images.configuration"
+      />.
+     </para>
+    </step>
+    <step>
+     <para>
+      Prepare ISO images containing the <command>cloud-init</command>
+      configuration files. See <xref
+      linkend="sec.deploy.preparation.disk_images.iso-image" />.
+     </para>
+    </step>
+    <step>
+     <para>
+      Create a copy of the downloaded disk image for each virtual
+      machine, naming them appropriately: for example,
+      <literal>caas-admin</literal>, <literal>caas-master</literal>,
+      <literal>caas-worker1</literal>, <literal>caas-worker2</literal>
+      and so on.
+     </para>
+    </step>
+    <step>
+     <para>
+      Boot and configure the &admin_node;. See <xref
+      linkend="sec.deploy.preparation.qcow2.admin" />.
+     </para>
+    </step>
+    <step>
+     <para>
+      Boot and configure the worker nodes. See <xref
+      linkend="sec.deploy.preparation.disk_images.worker" />.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.preparation.disk_images.vmware">
+   <title>Converting Images for &vmware; ESX and ESXi</title>
    <para>
-    Additionally, you need to make a separate copy of the downloaded VM
-    disk image for each VM. We suggest keeping the original download
-    elsewhere and making a fresh copy for each node, naming it
-    appropriately: for example, <literal>caas-admin</literal>,
-    <literal>caas-master</literal>, <literal>caas-worker1</literal>,
-    <literal>caas-worker2</literal> and so on.
+    Downloaded disk images need to be converted for usage with &vmware;
+    ESX and ESXi. Run the following command on the downloaded disk
```

Comment:
Run the following command **on ESXi host** on the downloaded disk... (as the command is available only on ESXi host and not SLE)